pro/src/new_components/StockThingEventForm/__specs__/validationSchema.spec.tsx

```diff
@@ -0,0 +1,217 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Formik } from 'formik'
+import React from 'react'
+
+import { IStockThingFormProps } from 'new_components/StockThingForm/StockThingForm'
+import { SubmitButton } from 'ui-kit'
+import { getToday } from 'utils/date'
+
+import { STOCK_THING_EVENT_FORM_DEFAULT_VALUES } from '../constants'
+import StockThingEventForm from '../StockThingEventForm'
+import { getValidationSchema } from '../validationSchema'
+
+jest.mock('utils/date', () => ({
+  ...jest.requireActual('utils/date'),
+  getToday: jest
+    .fn()
+    .mockImplementation(() => new Date('2022-12-25T00:00:00Z')),
+}))
+
+const today = getToday()
+const yesterday = getToday()
+const tomorrow = getToday()
+yesterday.setDate(yesterday.getDate() - 1)
+tomorrow.setDate(tomorrow.getDate() + 1)
+
+const renderStockThingEventForm = ({
+  minQuantity,
+}: {
+  minQuantity?: number | null
+} = {}) => {
+  const props: IStockThingFormProps = { today }
+  return render(
+    <Formik
+      initialValues={STOCK_THING_EVENT_FORM_DEFAULT_VALUES}
+      onSubmit={() => {}}
+      validationSchema={getValidationSchema(minQuantity)}
+    >
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <StockThingEventForm {...props} />
+          <SubmitButton>Submit</SubmitButton>
+        </form>
+      )}
+    </Formik>
+  )
+}
+
+describe('StockThingEventForm:validationSchema', () => {
+  it('test required fields', async () => {
+    renderStockThingEventForm()
+
+    await userEvent.click(screen.getByRole('button', { name: 'Submit' }))
+    expect(screen.getByTestId('error-eventDatetime')).toBeInTheDocument()
+    expect(screen.getByTestId('error-eventTime')).toBeInTheDocument()
+    expect(screen.getByTestId('error-price')).toBeInTheDocument()
+    expect(
+      screen.queryByTestId('error-bookingLimitDatetime')
+    ).not.toBeInTheDocument()
+    expect(screen.queryByTestId('error-quantity')).not.toBeInTheDocument()
+  })
+
+  const dataSetEventDateTime: Array<number> = [
+    today.getDate(),
+    tomorrow.getDate(),
+  ]
```

Comment:
fonctionne si on ajoute: 
```suggestion
  const dataSetEventDateTime: Array<number> = [
    yesterday.getDate(),
    today.getDate(),
    tomorrow.getDate(),
  ]
```

Alors que ca devrait casser.

Reply:
Les champs dateTime n'affichent pas d'erreur si on clique sur le jour précédent "today", il n'est pas sélectionnable, ça enlève le focus sur l'input, et lorsqu'on clique à côté, aucune erreur ne s'affiche.
En revanche sélectionner "today" devrait casser effectivement !